modern/src/runtime/WinampAbstractionLayer.js

```diff
@@ -1,6 +1,7 @@
 import Group from "./Group";
 
 // Needs to behavve like a group to work with top level scripts that call `getScriptGroup`
+/* eslint-disable-next-line local-rules/maki-methods */
```

Comment:
@jberg I forget the story of this one.

Reply:
I guess that WinampAbstraction layer isn't actually a MAKI object in the sense of being defined as a Class in `std.mi`, it's the XML wrapper at the top level. Should we give it a Js prefix?

Reply:
Ah, that makes sense. I’ll update the PR to have it rename it to use Js.